src/dotnet/updater.py

```diff
@@ -408,7 +398,7 @@ def _is_latest_dotnet(version: _DOTNET_VERSION_T, os_version: OsVersion) -> bool
                 version=ver,
                 name="dotnet-runtime",
                 is_sdk=False,
-                pretty_name=f".NET {ver} runtime",
+                pretty_name=f".NET Runtime {ver}",
```

Comment:
we use sentence style capitalization here and "runtime" suffix is used elsewhere. why do we need to change this?

Reply:
these are the official names and case styling used by Microsoft.

Reply:
@dmpop or @lvicoun can you advise what is best here?